solutions/src/main/java/ru/sberbank/school/task02/ConversionService.java

```diff
@@ -0,0 +1,85 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+
+public class ConversionService implements FxConversionService {
+
+    private ExternalQuotesService externalQuotesService;
+
+    public ConversionService(ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        try {
+            if(operation == null || symbol == null || amount.compareTo(BigDecimal.ZERO) < 0 || amount == null)
+                throw new IllegalArgumentException();
+        }
+        catch (IllegalArgumentException ex){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1223471292>*